app/src/main/java/com/github/polypoly/app/ui/game/TradeDialogsUIComponent.kt

```diff
@@ -0,0 +1,234 @@
+ * @param trade The trade to accept
+ * @param isApplicant A boolean to know if the player is the applicant or the receiver
+ */
+@Composable
+fun AcceptTradeDialog(trade: TradeRequest, isApplicant: Boolean, gameModel: GameViewModel) {
+    AlertDialog(
+        modifier = Modifier.testTag("accept_trade_dialog"),
+        onDismissRequest = {},
+        title = { Text(text = "Do you want to trade ${trade.locationGiven?.locationProperty?.name} against" +
+                " ${trade.locationReceived?.locationProperty?.name} with ${trade.playerApplicant.user.name}?") },
+        text = {},
+        buttons = {
+            Row (
+                modifier = Modifier
+                    .padding(Padding.medium)
+                    .fillMaxWidth()
+            ) {
+                Button(onClick = {
+                    if(isApplicant) {
+                        trade.currentPlayerApplicantAcceptation = true
+                        gameModel.updateTradeRequest(trade)
+                    } else {
+                        trade.currentPlayerReceiverAcceptation = true
+                        gameModel.updateTradeRequest(trade)
+                    }
+                }) {
+                    Text(text = "Accept")
+                }
+                Spacer(modifier = Modifier.width(Padding.medium))
+                Button(onClick = {
+                    if(isApplicant) {
+                        trade.currentPlayerApplicantAcceptation = false
+                        gameModel.updateTradeRequest(trade)
+                    } else {
+                        trade.currentPlayerReceiverAcceptation = false
+                        gameModel.updateTradeRequest(trade)
+                    }
+                }) {
+                    Text(text = "Refuse")
+                }
+            }
+        }
+    )
+}
+
+/**
+ * Dialog to inform the player that he/she has to wait for the other player decision
+ */
+@Composable
+fun WaitingForTheOtherPlayerDecisionDialog() {
+    AlertDialog(
+        modifier = Modifier.testTag("waiting_for_the_other_player_decision_dialog"),
+        onDismissRequest = {},
+        title = { Text(text = "Waiting for the other player decision ... ") },
+        text = {},
+        buttons = {}
+    )
+}
+
+/**
+ * Dialog to inform the player that the trade is done
+ * @param result The result of the trade (true if the trade is successful, false otherwise)
+ * @param openDialog A mutable state to open and close the dialog.
+ * @param gameModel The game model
+ * @param trade The trade that has been done
+ */
+@Composable
+fun TheTradeIsDoneDialog(result: Boolean, openDialog: MutableState<Boolean>?, gameModel: GameViewModel,
+                         trade: TradeRequest) {
+    AlertDialog(
+        modifier = Modifier.testTag("the_trade_is_done_dialog"),
+        onDismissRequest = {
+            openDialog?.value = false
+        },
+        title = {
+            if (result)
+                Text(text = "The trade is done !")
```

Comment:
hardcoded text

Reply:
I created my PR before the merging of the PR of @jadtl that fix the hardcoded text, this is the reason